client/components/Common/Pagination.tsx

```diff
@@ -1,47 +1,103 @@
 import React from 'react'
 import { Pagination, PaginationItem, PaginationLink } from 'reactstrap'
 
+// it is the same as server's definition
+export interface Pager {
+  page: any
+  pagesCount: number
+  pages: number[]
+  total: number
+  previous: number | null
+  previousDots: boolean
+  next: number | null
+  nextDots: boolean
+}
```

Comment:
Regarding using these types, I think the server-side pager object is for twig template, not for React components.
In other words, I'd like to make the pager type more simple when we remove `views/widget/pager`. But I have return pager object for React components when I migrated admin pages😇

BTW, Would you like to stop creating pagination from the pager object? I think the view itself that the new component renders is so great.

Reply:
Alright, it should be on React.
(In this PR, I didn't remove the pagination logic on server side though ...)